include/libhal-input/ch9329_kb_bytes.hpp

```diff
@@ -0,0 +1,172 @@
+// Copyright 2024 Khalil Estell
```

Comment:
Can we change the name of the file from `include/libhal-input/ch9329_kb_bytes.hpp` to `include/libhal-input/ch9329_keyboard_constants.hpp` Its long but it explains whats going on.